src/lib/validation/utils/assignHandlerForTrigger.js

```diff
@@ -44,10 +45,19 @@ export default function assignHandlerForTrigger (triggerButton, form) {
       invalidCustomErrorFields.forEach(field => {
         dplan.notify.notify('error', Translator.trans(field.validationMessage))
       })
-      // Make sure the default notification is displayed as needed
+      const nonEmptyFieldNames = [...new Set(
+        validatedForm.invalidFields
+          .map(field => field.getAttribute('data-dp-validate-error-fieldname'))
+          .filter(Boolean)
+      )]
+
       if (invalidCustomErrorFields.length === 0 || invalidCustomErrorFields.length < validatedForm.invalidFields) {
-        dplan.notify.notify('error', Translator.trans('error.mandatoryfields.no_asterisk'))
+        const fieldsString = nonEmptyFieldNames ? nonEmptyFieldNames.join(', ') : ' '
+        const errorMandatoryFields = de.error.mandatoryFields.intro + fieldsString + de.error.mandatoryFields.outro
```

Comment:
Are the trans-keys defined somewhere? Can't find them. (only another usage, but no declaration)
```suggestion
        const errorMandatoryFields = `${de.error.mandatoryFields.intro} ${fieldsString} ${de.error.mandatoryFields.outro}`
```

Reply:
yes, they are defined here --> '~/components/shared/translations'

Reply:
sorry. I am still used to find transkeys when searching for the whole string.